src/block_implementations/Eos/EosFaderBankBlock.cpp

```diff
@@ -45,6 +48,7 @@ void EosFaderBankBlock::setPageFromGui(int value) {
     emit faderLevelsChanged();
     sendConfigMessage();
     emit pageChanged();
+    setValue(m_page);
```

Comment:
For what do you use the new output node? Please add a hint to the helpText what it can be used for (maybe MIDI feedback on the page?).

Reply:
yes, it is for possible feedback of actual page, however no real usage yet